cms/static/js/views/uploads.js

```diff
@@ -43,8 +43,8 @@ define(['jquery', 'underscore', 'gettext', 'js/views/modals/base_modal', 'edx-ui
                 // this if the selected file is valid: if it isn't, we want to render
                 // a blank input to prompt the user to upload a different (valid) file.
                 if (selectedFile && isValid) {
-                    $(oldInput).removeClass('error');
-                    this.$('input[type=file]').replaceWith(HtmlUtils.ensureHtml(oldInput).toString());
+                    oldInput.removeClass('error');
```

Comment:
Works fine and the tests also pass when I do this instead:
`$(oldInput).removeClass('error');`

Reply:
`oldInput` is jquery object and when you wrap an object inside `$` its still an object. See below. 
<img width="822" alt="Screen Shot 2020-09-02 at 11 55 25 PM" src="https://user-images.githubusercontent.com/4252738/92024571-c8901b80-ed77-11ea-971a-62b95e85fc1f.png">